docs: for review Starport docs v0.14 (#852)
tendermint/starport: docs/1 Introduction/3 Quickstart.md

# Quickstart

> Use these three commands to build, start, and define your first blockchain.

Reviewer: ```suggestion
Use these three commands to create, start, and add functionality to your first blockchain.
```

## Create a blockchain

To create a blockchain:

```

> starport app github.com/username/myapp && cd myapp

Reviewer: ```suggestion
starport app github.com/hello/planet && cd planet
```

```

> The `app` command creates the blockchain directory `myapp` and scaffolds a [Cosmos SDK](https://docs.cosmos.network/) blockchain.

Reviewer: ```suggestion
The `app` command creates the blockchain directory `planet` and scaffolds a [Cosmos SDK](https://docs.cosmos.network/) blockchain.
```

## Run a blockchain

> To run a blockchain in your development environment:

Reviewer: ```suggestion
To run a blockchain node in your development environment:
```

```
starport serve
```

The `serve` command installs dependencies, builds, initializes, and starts the blockchain.

> ## Define

Reviewer: ```suggestion
## Add functionality
```

To add a custom type with create, read, update, and delete (CRUD) functionality:

```
starport type post title body
```

> The `type` command scaffolds functionality a custom type.

Reviewer: ```suggestion
The `type` command scaffolds functionality for a custom type.
```

> 

Reviewer: ## Next steps

To learn more about building Cosmos SDK blockchains with Starport, try a beginner-friendly [IBC Hello world tutorial](http://tutorials.cosmos.network/hello-world/tutorial/).

## Learn more

- [Configure a Blockchain](./3%20Configure%20Your%20Blockchain)
- [Run a Blockchain](./2%20Run%20a%20Blockchain)
- [Starport repo in GitHub](https://github.com/tendermint/starport)
- [Cosmos SDK Documentation](https://docs.cosmos.network)
- [Cosmos SDK Tutorials](https://tutorials.cosmos.network)
- [Starport Channel on Discord](https://discord.com/channels/669268347736686612/737461683588431924)

> <!-- link to starport tutorial for 2 blockchains? where do we send our users next -->

Reviewer: ```suggestion
```
